host_core/lib/host_core/claims/manager.ex

```diff
@@ -88,8 +89,15 @@ defmodule HostCore.Claims.Manager do
       contract_id: Map.get(claims, :contract_id) || ""
     }
 
-    cache_claims(lattice_prefix, public_key, claims)
-    publish_claims(host_id, lattice_prefix, claims)
+    newclaims =
+      if claims.config_schema != nil do
+        Map.put(newclaims, :config_schema, claims.config_schema)
+      else
+        newclaims
+      end
```

Comment:
Would something bad happen if we did `Map.put(newclaims, :config_schema, claims.config_schema)` even when the schema is nil? Is there a functional difference between "key is present but value is nil" and "key is not present" in this case?

Reply:
Yeah if you do `Map.put(bob, :alice, nil)` you get a map with the key of alice and the value of `nil`.
If you don't put the key at all, you leave the key out of the map. I figured it was just less clutter in the claims cache if we leave that key out if we don't have a schema. All the code that reads from the schema checks for the existence of the key, it won't panic on a nil

Reply:
in other words, I just didn't want to fill all the claims with `"config_cache": nil` everywhere